Examples/Scripts/Python/particle_gun.py

```diff
@@ -17,18 +17,54 @@
 from acts import Vector4, UnitConstants as u
 
 
-def runParticleGun(outputDir, s=None):
+def runParticleGun(
+    outputDirCsv = None,
+    outputDirRoot = None, 
+    pConfig = [1 * u.GeV, 10 * u.GeV, True],
+    etaConfig = [-4., 4., True],
+    particleConfig = [10, acts.ActsPythonBindings.PdgParticle.eMuon, True],
+    vtxGen = None, 
+    s=None
+    ):
+    """ This function steers the particle generation using the particle gun
+    
+        Parameters
+        ----------
+        outputDirCsv : str, path, None
+            the output folder for the Csv output, None triggers no output
+        outputDirRoot : str, path, None
+            the output folder for the Root output, None triggers no output
+        pConfig : list
+            momentum configuration: minimum momentum, maximum momentum, transverse
+        etaConfig : list
+            pseudorapidity configuration: eta min, eta max, uniform
+        particleConfig: list
+            partilce configuration: number of particles, particle type, charge flip
```

Comment:
Argubaly this could be separate arguments, like `numParticles`, `particlePdg` and `particleCargeFlip` or something.